Microsoft.Toolkit.Uwp.UI.Controls/UniformGrid/UniformGrid.Helpers.cs

```diff
@@ -73,7 +73,7 @@ internal static IEnumerable<(int row, int column)> GetFreeSpot(TakenSpotsReferen
         // returns the dimensions of the
         // grid we need to hold all elements.
         #pragma warning disable SA1008 // Opening parenthesis must be spaced correctly
-        internal static (int rows, int columns) GetDimensions(ref IEnumerable<FrameworkElement> visible, int rows, int cols, int firstColumn) //// TODO: Switch to 'in' for C# 7.2
+        internal static (int Rows, int Columns) GetDimensions(FrameworkElement[] visible, int rows, int cols, int firstColumn)
```

Comment:
I think the standard is lowercase. for Tuple properties.

Reply:
Is there a coding standard for this yet? My thought is if you do `var dim = GetDimensions` then you would want the properties to be upper case `dim.Row`. But, if you extract as you suggested above `var (rows, columns) = GetDimensions` it makes more sense to leave them lower case

Reply:
Not sure if there is a standard but I have always seen Tuple deconstruction used with lowercase. Either that or use a Struct instead of a Tuple.